storage/indexer/src/db_indexer.rs

```diff
@@ -155,7 +173,7 @@ impl DBIndexer {
         let mut db_iter = self.get_main_db_iter(version, num_transactions)?;
         let batch = SchemaBatch::new();
         db_iter.try_for_each(|res| {
-            let (txn, events) = res?;
+            let (txn, events, writeset) = res?;
             if let Some(txn) = txn.try_as_signed_user_txn() {
```

Comment:
hmm.. In AptosDB I don't think we skip event & state indexing for non-user txns?